.travis.yml

```diff
@@ -40,33 +27,149 @@ before_install:
   - $GRADLE_SCRIPT --version
   - groovy --version
   - sudo chmod a+rX $HOME
-# We don't currently use these sources, but they can be re-enabled if necessary
-#  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv 3375DA21
-#  - echo deb http://apt.thehyve.net/internal/ trusty main | sudo tee /etc/apt/sources.list.d/hyve_internal.list
-#  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv E5267A6C
-#  - echo deb http://ppa.launchpad.net/ondrej/php/ubuntu trusty main  | sudo tee /etc/apt/sources.list.d/ondrej_php.list
   - sudo apt-get -qq update
   - sudo apt-get install -y tk-dev html2text gnupg2
   - gpg2 --version
   - php --version
-  - ([ -z "$PREINSTALL" ] && echo "Skipping preinstall.") || (echo $PREINSTALL; $PREINSTALL)
   - PWD=`pwd`
   - echo $PWD
 
-install:
-  - pushd "$DIR"
-  - ([ -z "$INSTALL" ] && echo "Skipping install.") || (echo $INSTALL; $GRADLE_SCRIPT $INSTALL)
-  - popd
+jobs:
+  include:
+    - stage: test
+      name: transmart-core-api
+      env: DIR=transmart-core-api
+      script:
+        - pushd "$DIR"
+        - $GRADLE_SCRIPT check
+        - popd
+
+    - name: transmart-core-db-tests
+      env: DIR=transmart-core-db-tests
+      before_script:
+        - pushd "$DIR"
+        - $GRADLE_SCRIPT assemble
+        - popd
+      script:
+        - pushd "$DIR"
+        - $GRADLE_SCRIPT check
+        - popd
+
+    - name: transmart-rest-api
+      env: DIR=transmart-rest-api
+      script:
+        - pushd "$DIR"
+        - $GRADLE_SCRIPT check
+        - popd
+
+    - name: transmart-core-db on PostgreSQL 9.6
```

Comment:
9.6 or 9.2? Or it is not related with postgresql-9.2 that is removed later?

Reply:
`9.2` is default, but we configured `9.6` to be used instead. Up to version `10`, choosing a database version is relatively easy, for `11` you need to do more effort, such as actually removing `9.2`.